config/software/gdbm.rb

```diff
@@ -44,6 +44,7 @@
 
   configure_command = ["./configure",
                        "--enable-libgdbm-compat",
+                       "--without-readline",
```

Comment:
To reviewer: added because I was facing this error on osx:

```
  CC       var.o
  CC       util.o
  CC       wordwrap.o
  CC       input-rl.o
input-rl.c:194:12: error: implicit declaration of function 'history_list' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    return history_list ()[n]->line;
           ^
input-rl.c:194:12: note: did you mean 'history_get'?
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/readline/history.h:153:13: note: 'history_get' declared here
HIST_ENTRY      *history_get(int);
                 ^
input-rl.c:194:27: error: subscripted value is not an array, pointer, or vector
    return history_list ()[n]->line;
           ~~~~~~~~~~~~~~~^~
```

I stole the fix from the homebrew formula and it worked well for me: https://github.com/Homebrew/homebrew-core/blob/d48ec91040f4c3deb013d734f89897ad56d4357b/Formula/gdbm.rb

Reply:
I think readline would only be required by the files in `bin` to improve user experience, so I think this is perfectly fine.